grafana/dashboard.json

```diff
@@ -0,0 +1,1065 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": "-- Grafana --",
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
```

Comment:
Was able to sucessfuly import this configuration in grafana. I need to export data from DD to verify that it's working well, might need a bit of help on that

Reply:
Ok I was able to test it and it is working well!